ios/Classes/Common/FlutterMethodCallHandler.h

```diff
@@ -0,0 +1,7 @@
+#import <Flutter/Flutter.h>
+
+@protocol FlutterMethodCallHandler
+
+- (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result;
```

Comment:
We don't want FlutterBleLib and BLEmulator to have `Common`, since the latter is to be extracted to a separate repository.

Reply:
This is a temporary solution until Blemulator is extracted from the repo. My plan was to place all "Common" code here so I can know later which classes I'm going to need to place in both repos.

Reply:
Created a separate issue for that to address that after currently opened PRs are merged